src/Illuminate/Console/Scheduling/Event.php

```diff
@@ -915,4 +930,16 @@ public function preventOverlapsUsing(EventMutex $mutex)
 
         return $this;
     }
+
+    /**
+     * Delete the mutex for the event.
+     *
+     * @return void
+     */
+    protected function deleteMutex()
```

Comment:
@taylorotwell The reason I called this `removeMutex` is because that's what it's already called in `CallbackEvent`. It probably either needs to be renamed in both places or left as `removeMutex`.

https://github.com/laravel/framework/blob/10483fbd04643135c242850c20467e3c182dce17/src/Illuminate/Console/Scheduling/CallbackEvent.php#L101-L106

Reply:
OK we rename it back to removeMutex